templates/notifiers.html

```
@@ -1,22 +1,15 @@
 {% extends "base.html" %}
 
 {% block title %}Notifiers - File Fridge{% endblock %}
+{% block page_title %}Notifiers{% endblock %}
+{% block page_description %}Configure email and webhook destinations for system alerts and operational notifications with responsive editing controls.{% endblock %}
+{% block page_actions %}
+<button class="btn btn-primary" id="create_notifier_btn">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)